engine/immediate_task_queue.go

```diff
@@ -68,14 +68,21 @@ func NewImmediateTaskQueueImpl(
 
 func (w *immediateTaskQueueImpl) Stop(ctx context.Context) error {
 	// close timer to prevent goroutine leakage
-	w.pollTimer.Close()
```

Comment:
I found timer.close() will cause error, need to use stop(). Though don't know why.

Reply:
Tried in the last two commits, adding close() cause bugs.